src/widgets/BannerCarousel/ui/Carousel.tsx

```
@@ -0,0 +1,134 @@
+  font-weight: 700;
+  margin-left: 30px;
+  padding: 3px 10px;
+  border: 1px solid ${({ theme }) => theme.colors.accent};
+  border-radius: 16px;
+  background-color: white;
+  color: ${({ theme }) => theme.colors.accent};
+  white-space: nowrap;
+  flex-shrink: 0;
+`;
+const Date = styled.p`
+  font-size: 0.75rem;
```

Biome (1.9.4)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)